homeassistant/components/lg_netcast/strings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Ensure that your TV is turned on before trying to set it up.\nIf you leave the host empty, discovery will be used to find devices.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
```

Comment:
Let`s add host filed description.

Example:
https://github.com/home-assistant/core/blob/e0490a3ade04fdc3fcbcacb16ce56ad308408f31/homeassistant/components/braviatv/strings.json#L6-L11

Reply:
This has been added